Gemfile.lock

```diff
@@ -307,10 +307,6 @@ GEM
     sax-machine (1.3.2)
     sentry-raven (2.7.1)
       faraday (>= 0.7.6, < 1.0)
-    shoulda (3.5.0)
```

Comment:
Interesting, I always thought that shoulda-matchers would depend on it. The more you know :) Good bye superfluous dependency :wave:

Reply:
I've never used `shoulda`, but I think it adds some `RSpec` like macros to `minitest`. So you probably need it for the "should" part if you use these matchers with `minitest`.
For RSpec, we won't need it.